tests/devices/qubit/test_preprocess.py

```diff
@@ -594,24 +594,6 @@ def test_valid_tape_with_expansion(self, shots):
 class TestPreprocess:
     """Unit tests for ``qml.devices.qubit.preprocess``."""
 
-    def test_choose_best_gradient_method(self):
```

Comment:
should these tests be re-added elsewhere?

Reply:
moved to just being in the device tests.